### PR TITLE
Add CHANGELOG

### LINKED ISSUE #15: add CHANGELOG
cf #10

### PR DESCRIPTION
**Pull request by MaxUlysse**
- add up to date `CHANGELOG` closes #15 
- add `.travis.yml` but still need to set up actual tests, but that will be for @andreas-wilm cf #1 
- remove institute logos

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,14 +35,11 @@ Documentation about the pipeline can be found in the `docs/` directory:
...
-
 ### Credits
 
-This pipeline was originally developed at the
-[Genome Institute of Singapore](http://a-star.edu.sg/gis) by
-[Andreas Wilm](https://github.com/andreas-wilm/). It started out as an
-ecosystem around LoFreq an went through a couple of iterations. The
-current version had three predecessors
+This pipeline was originally developed by Andreas Wilm ([andreas-wilm](https://github.com/andreas-wilm)) at [Genome Institute of Singapore](https://www.a-star.edu.sg/gis/).
+It started out as an ecosystem around LoFreq an went through a couple of iterations.
```
Comment (apeltzer) [inline README.md]: and went through a couple of iterations.

**Comment by apeltzer** [comment]
``` 
mkdir: cannot create directory ‘/home/travis/build/nf-core/vipr/tests’: File exists
```

**Comment by MaxUlysse** [comment]
I think we have to make some tests before trying to fix them

**Comment by ewels** [comment]
Tests are failing because the repository already has a directory called `tests`. It looks like it all needs to be removed - it's old snakemake files and stuff. So yeah, fixing this is a larger job. Now that the tests run we will find it easier to move towards a validated pipeline though 👍

**Pull request merged**